bestPractices.js

```diff
@@ -0,0 +1,333 @@
+const loadElements = () => {
+  let taskArray = [];
+  taskArray = retrieveData();
```

Comment:
- Would be great if remove 'let taskArray = []' on line 2 and declare and set the value in the same time to follow DRY principles like this: let taskArray = retrieve data();